pkg/kubelet/apis/config/types.go

```diff
@@ -444,6 +444,10 @@ type KubeletConfiguration struct {
 	// registerNode enables automatic registration with the apiserver.
 	// +optional
 	RegisterNode bool
+
+	// SkipCadvisorProcessMetrics provide a way to skip the cadvisormetrics.ProcessMetrics
```

Comment:
Can we document why someone might want to skip these metrics?

Reply:
Can you document what cadvisor/summary API metrics will be missing as a result?

Reply:
Of course. 

	// SkipCadvisorProcessMetrics provide a way to skip the cadvisormetrics.ProcessMetrics, as there is a performance issue when there is a huge number of open files. 
        // See https://github.com/kubernetes/kubernetes/issues/99183 for more details.
        // When skipping process metrics,  the cadvisor metrics would not include container_processes, container_file_descriptors, container_sockets, container_threads_max, container_threads, container_ulimits_soft. 
```